flask_selfdoc/autodoc.py

```diff
@@ -153,7 +254,7 @@ def generate(self, groups='all', sort=None):
                     methods=sorted(list(rule.methods)),
                     rule="%s" % rule,
                     endpoint=rule.endpoint,
-                    docstring=func.__doc__,
+                    docstring=func.__doc__.strip(' ') if func.__doc__ else None,
```

Comment:
I thought this could be more practical for teams where some developers have an IDE that automatically strips the trailing whitespaces